apps/greencheck/domain_check.py

```diff
@@ -228,13 +256,21 @@ def check_domain(self, domain: str) -> SiteCheck:
         the best matching IP range for the ip address it resolves to,
         or a 'grey' Sitecheck
         """
+        UNRESOLVED_ADDRESS = "0.0.0.0"
```

Comment:
I'm aware that 0.0.0.0 has a special meaning when we pass it to servers. I'm using this placeholder, as I'm not aware of an equivalent IP address that means "not a real address", and in a few places we always assume an ip address string is returned.